src/credentials/CHIPOperationalCredentials.h

```diff
@@ -219,6 +234,22 @@ class DLL_EXPORT OperationalCredentialSet
     CHIP_ERROR SetDevOpCred(const CertificateKeyId & trustedRootId, const uint8_t * chipDeviceCredentials,
                             uint16_t chipDeviceCredentialsLen);
 
+    /**
+     *  @brief
+     *    Serialize the OperationalCredentialSet indexed by a TrustedRootID to the given serializable data structure
+     *
+     *    This method must be called while the OperationalCredentialSet class is valid (After Init and before Release)
+     */
+    CHIP_ERROR ToSerializable(const CertificateKeyId & trustedRootId, OperationalCredentialSerializable & output);
+
+    /**
+     *  @brief
+     *    Reconstruct OperationalCredentialSet class from the serializable data structure.
+     *
+     *    This method must be called after initializing the OperationalCredentialSet class with internal allocation
```

Comment:
OK, but what about the input `OperationalCredentialSerializable`?  Can it go away after this call?  Or will the set end up with pointers into it?

Reply:
It can be freed after this call, updated comments to reflect this.